Makefile

```diff
@@ -23,6 +23,10 @@ start-cron: start-deps
 	. ./.envrc && yarn tsnd --files -r tsconfig-paths/register -r src/services/tracing.ts \
 		src/servers/cron.ts | yarn pino-pretty -c -l
 
+start-loop:
+	chmod +x ./scripts/start-loop.sh
```

Comment:
you can avoid this pushing the file with the correct permissions

Reply:
fixed